assembl/static2/css/components/text.scss

```diff
@@ -59,8 +59,9 @@
 .dark-title-5 {
   color: $title5-dark-color;
   font-family: $font-family4;
-  font-size: $title5-size;
+  font-size: $title5-size + 2px;
```

Comment:
There is no $font-size variable to get the same result ?

Reply:
nope. Should I create a new one or it's better keeping the css this way ?

Reply:
You should create a new class